### PR TITLE
[FIX][AGW]Fix post install sctpd

### PR DESCRIPTION
**Pull request by tmdzk**
[FIX][AGW]Fix post install sctpd

## Summary

Wrong naming in checking of sctpd service 

## Test Plan

./agw_post_install.sh

### CODE REVIEW

File: lte/gateway/deploy/agw_post_install.sh [hunk 1]
```diff
@@ -43,13 +43,20 @@ if [ "$PING_RESULT" != "$SUCCESS_MESSAGE" ]; then
...
 
-allServices=("control_proxy" "directoryd" "dnsd" "enodebd" "magmad" "mme" "mobilityd" "pipelined" "policydb" "redis" "sctpd" "sessiond" "state" "subscriberdb")
+allServices=("control_proxy" "directoryd" "dnsd" "enodebd" "magmad" "mme" "mobilityd" "pipelined" "policydb" "redis" "sessiond" "state" "subscriberdb")
 for service in "${allServices[@]}"; do
     if ! systemctl is-active --quiet "magma@$service"; then
         addError "$service is not running" "Please check our faq"
     fi
 done
 
+notMagmaServices=("sctpd")
```
Comment (ssanadhya) [inline lte/gateway/deploy/agw_post_install.sh]: Sctpd is still a Magma project service, it is just packaged separately than other services, and named differently. Can we call this set magmaSctpdServices as they are part of a different package?

**Review by ssanadhya — CHANGES_REQUESTED** [review]

File: lte/gateway/deploy/agw_post_install.sh (hunk 1 above)
Comment (tmdzk) [reply lte/gateway/deploy/agw_post_install.sh]: I wanted a name for all the services that are not managed by magmad

Comment (ssanadhya) [reply lte/gateway/deploy/agw_post_install.sh]: Ah, then let's do
```suggestion
nonMagmadServices=("sctpd")
```

Comment (tmdzk) [reply lte/gateway/deploy/agw_post_install.sh]: sounds good to me thanks for the great name :)

File: lte/gateway/deploy/agw_post_install.sh [hunk 2]
```diff
@@ -43,13 +43,20 @@ if [ "$PING_RESULT" != "$SUCCESS_MESSAGE" ]; then
...
-allServices=("control_proxy" "directoryd" "dnsd" "enodebd" "magmad" "mme" "mobilityd" "pipelined" "policydb" "redis" "sctpd" "sessiond" "state" "subscriberdb")
+allServices=("control_proxy" "directoryd" "dnsd" "enodebd" "magmad" "mme" "mobilityd" "pipelined" "policydb" "redis" "sessiond" "state" "subscriberdb")
 for service in "${allServices[@]}"; do
     if ! systemctl is-active --quiet "magma@$service"; then
         addError "$service is not running" "Please check our faq"
     fi
 done
 
+nonMagmadServices=("sctpd")
+for service in "${notMagmaServices[@]}"; do
```
Comment (ardzoht) [inline lte/gateway/deploy/agw_post_install.sh]: `nonMagmadServices`

**Review by ardzoht — APPROVED** [review]

**Pull request merged**